package-testing/python-sdk-relay/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.12
+
+WORKDIR /app
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt
+
+# Copy the source code
+COPY src/ ./src/
+
+CMD ["python", "/app/src/server.py"]
```

Comment:
We do the Python stuff because we're using the Python _test_ server to test the relay servers, correct?